frontend/style/mediaelementplayer/mediaelementplayer.css

```diff
@@ -0,0 +1,774 @@
+/* Accessibility: hide screen reader texts (and prefer "top" for RTL languages).
```

Comment:
Should be better to have the full size JS and CSS and let grunt minified all the files in the build?

Because when you have to debug is very difficult to look at.

Reply:
For the moment I will delete it. :)

Reply:
Optimally, we would install our dependencies via `npm` and glue them together using our build system (which, for now, is `grunt`, yes, but optimally would not be `grunt` :wink:).

Including non-minified files for debugging purposes is fine for now, I guess, but we should not have two versions of a library. Also I prefer having such large changes in a separate PR.

Reply:
Ok, no problem.